### DISCUSSION TITLE
Does payload have list reactive?

### DISCUSSION THREAD

**Discussion by pttsync** [payloadcms/payload#934] [Q&A] [answered]
Suppose I have 2 user.
User A open a posts listing page. When user B create new post, User A will see that post added to posts listing page imediately without refresh the page. Is this possible?

**Comment by denolfe** [comment, marked as answer]
Hey @pttsync , unfortunately, the current admin UI does not have the ability to reflect new documents without a refresh or page change. Feel free to make a new discussion under "Feature Requests & Ideas" with what you're looking to see, so we can track the interest from the community via upvotes.

If you're looking to get really in the weeds, this could possibly be done if you swap out the default list view with a customized one that has the functionality you need - maybe polling the API on an interval. Docs regarding that [here](https://payloadcms.com/docs/admin/components#collections)
